src/sensors/SensorManager.h

```diff
@@ -56,12 +57,12 @@ namespace SlimeVR
 
             void update();
             
-            Sensor  ** const getSensors() { return m_Sensors; };
+            std::vector<Sensor *> & getSensors() { return m_Sensors; };
```

Comment:
We should probably use smart pointers for this, but again, I'm no C++ expert
Here's some MS docs about them: https://learn.microsoft.com/en-us/cpp/cpp/smart-pointers-modern-cpp?view=msvc-170#c-standard-library-smart-pointers